PythonTool/DataLoading.py

```diff
@@ -0,0 +1,208 @@
+
+   Returns:
+   ________
+      new_file_name: str
+          The new file name with the specified extension.
+   """
+
+   # Load the file to be converted.
+   mb = core.Core()
+   mb.load_file(file_location)
+
+   # Isolate file name from string containing the file location.
+   input_array = file_location.split("/")
+   input_file = input_array[-1]
+   base_file_name = input_file[:-4]
+   new_file_name = base_file_name + file_extension
+
+   # Write the new file with the user supplied extension.
+   mb.write_file(new_file_name)
+
+   return(new_file_name)
+
+
+def plane_slice_plotting(window_number, axis_number, label, images):
+    """
+    Copy the Mesh, Pseudocolor, and Contour plots into a new VisIt window and
+    slice through the proper axis.
+
+    Input:
+    ______
+      window_number: int
+          The number of the window to open (2,3, or 4).
+      axis_number: int
+          The number of the axis to slice through (0 for X, 1 for Y, 2 for Z).
+      images: boolean
+          Whether or not to save images of the plot windows.
+
+    Returns:
+    ________
+      none
+    """
+
+    # Open a new window with all three plots active.
+    Vi.AddWindow()
+    Vi.CopyPlotsToWindow(1, window_number)
+    Vi.SetActivePlots((0,1,2))
+
+    # Add a slice through the proper axis.
+    Vi.AddOperator("Slice", 1)
+    s = Vi.SliceAttributes()
+    s.axisType = axis_number
+    Vi.SetOperatorOptions(s)
+
+    banner = Vi.CreateAnnotationObject("Text2D")
+    banner.position = (0.37, 0.92)
+    banner.text = label
+    banner.height = 0.03
+
+    Vi.DrawPlots()
+    if images:
+        Vi.SaveWindow()
+
+
+def data_loading(geometry_file, data_file, images):
+   """
+   Convert geometry file to stl, convert data file to vtk, load each file
+   into VisIt, and display four interactive plot windows.
+       1) A cube with a slice through an octant in order to see the center.
+       2) XY plane slice through the center.
+       3) XZ plane slice through the center.
+       4) YZ plane slice through the center.
+   Each window has a mesh plot with the "STL_mesh" variable, a Pseudocolor plot
+   with the "TALLY_TAG" variable, and the second, third, and fourth windows have
+   Contour plots with the "ERROR_TAG" variable.
+
+   Input:
+   ______
+      geometry_file: h5m file
+          User supplied file containing geometry of interest.
+      data_file: h5m or vtk file
+          User supplied file containing data of interest.
+      images: boolean
+          Whether or not to save images of the plot windows.
+
+   Returns:
+   ________
+      none
+   """
+
+   # Remove the graveyard from the geometry file.
+   remove_graveyard(geometry_file)
+
+   # Isolate the name of the geometry file produced from removing the graveyard.
+   input_array = geometry_file.split("/")
+   input_file_name = input_array[-1]
+   base_file_name = input_file_name[:-4]
+   geometry_file = base_file_name + "_no_grave.h5m"
```

Comment:
We'd talked about either returning the name of the file created by `remove_graveyard`. Is that slated for a future PR?

Reply:
I have included that functionality in my most recent commit!